codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -119,8 +127,51 @@ If there are obsolete binaries in entitlements configuration files, please delet
 update these file paths accordingly.
 ''';
 
+  /// Extract credentials needed for code sign app.
```

Comment:
We can get rid of most of this complexity using a file per secret.

Reply:
Sure, updated to one secret per file.